docs/output.md

```diff
@@ -279,39 +284,69 @@ Results generated by MultiQC collate pipeline QC from supported tools e.g. FastQ
   - `*.bai`: Index of the corresponding bam file
   </details>
 
+### Repeat annotation
+
+[Stranger](https://github.com/Clinical-Genomics/stranger) is used to annotate repeats.
+
+<details markdown="1">
+<summary>Output files from Stranger</summary>
+
+- `{outputdir}/repeat_annotation/stranger/{sample}`
+  - `*.vcf.gz`: Annotated VCF
+  - `*.vcf.gz.tbi`: Index of the corresponding VCF file
+  </details>
+
 ### SNV Annotation
 
 [echtvar](https://github.com/brentp/echtvar) and [VEP](https://www.ensembl.org/vep) is used to annotate small variants.
+In case of affected samples, [echtvar](https://github.com/brentp/echtvar) and [VEP](https://www.ensembl.org/vep) is used to annotate small variants.
```

Comment:
```suggestion
In case of affected samples, [echtvar](https://github.com/brentp/echtvar) and [VEP](https://www.ensembl.org/vep) are used to annotate small variants.
```